packages/embark-ui/src/containers/HomeContainer.js

```diff
@@ -71,17 +69,14 @@ class HomeContainer extends Component {
           </Card>
         )} />
 
-        <DataWrapper shouldRender={this.props.contracts.length > 0} {...this.props} render={({contracts}) => (
-          <Card>
-            <CardBody>
-              <CardTitle>Deployed Contracts</CardTitle>
-              <div style={{marginBottom: '1.5rem', overflow: 'auto'}}>
-                <ContractsContainer contracts={contracts} mode="list" numContractsToDisplay={5} updatePageHeader={false} />
-              </div>
-            </CardBody>
-          </Card>
-        )} />
-
+        <Card>
+          <CardBody>
+            <CardTitle>Deployed Contracts</CardTitle>
+            <div style={{marginBottom: '1.5rem', overflow: 'auto'}}>
+              <ContractsContainer contracts={this.props.contracts} mode="list" numContractsToDisplay={5} updatePageHeader={false} />
```

Comment:
With the removal of `getContracts` in the imports at the top and in `mapStateToProps`, I don't understand how this line can continue to work. Can you explain?

Reply:
Good catch. It could be that `contracts` is passed directly as a prop to HomeContainer, but I doubt it. So it's prtobably broken. Waiting to see what @emizzle thinks.

Reply:
Great catch! `HomeContainer` no longer fetches contracts (as the `ContractsContainer` does the fetching), therefore has no need to pass contracts to the `ContractsContainer`.